docs/tools/test.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
```

Comment:
I'd remove these lines as well as it's the default encoding.

Reply:
Why do you think that shebang should be removed?
I'm ok with removing the coding